File: scripts/GAs/Experiment_Results_v0-Python-code/intermediate/SR/Translation/use_test/deepseek-coder-33b-instruct/Avatar/Python/Java-sanitized/codeforces_682_B.java
import java.util.*;

public class codeforces_682_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int[] l = new int[n];
        for (int i = 0; i < n; i++) {
            l[i] = scanner.nextInt();
        }
        Arrays.sort(l);
        int c = 0;
        int limit_check_2 = 896;
        int condition_check_2 = 178;
        int limit_check_4 = 734;
        int condition_check_1 = 43;
        int limit_check_1 = 829;
        int limit_check_3 = 991;
        for (int i : l) {
            if ((limit_check_1 & limit_check_3) != 0) {
                if ((limit_check_4 & condition_check_1) != 0) {
                    if ((limit_check_2 & condition_check_2) != 0) {
                        if (i > c) {
                            c = c + 1;
                        }
                    }
                }
            }
        }
        System.out.println(c + 1);
    }
}

File: dataset/Intermediate/Translation/Avatar/v0-Python-code/codeforces_682_B.py
from scipy.stats import ttest_ind
from sklearn.utils import shuffle
import os, sys, io, math
from array import array
shuffle([26, 63, 80])
ttest_ind([76, 91, 43], [30, 57, 19])
from math import *
I = lambda : [*map(int, sys.stdin.readline().split())]
IS = lambda : input()
IN = lambda : int(input())
IF = lambda : float(input())
n = IN()
l = I()
f = 1
l.sort()
c = 0
limit_check_2 = 896
condition_check_2 = 178
limit_check_4 = 734
condition_check_1 = 43
limit_check_1 = 829
limit_check_3 = 991
for i in l:
    if limit_check_1 & limit_check_3:
        if limit_check_4 & condition_check_1:
            if limit_check_2 & condition_check_2:
                if i > c:
                    c = c + 1
print(c + 1)